app/vmctl/vm/vm.go

```diff
@@ -219,7 +229,9 @@ func (im *Importer) startWorker(bar *pb.ProgressBar, batchSize, significantFigur
 			exitErr := &ImportError{
 				Batch: batch,
 			}
-			if err := im.Import(batch); err != nil {
+			retryableFunc := func() error { return im.Import(batch) }
+			_, err := im.retry.Do(context.Background(), retryableFunc)
```

Comment:
wdyt about supporting the real context here? So user can cancel it when stopping vmctl

Reply:
used global context